packages/java-parser/test/samples-spec.js

```diff
@@ -10,6 +10,24 @@ const javaParser = require("../src/index");
 describe("The Java Parser", () => {
   createSampleSpecs("java-design-patterns");
   createSampleSpecs("spring-boot");
+  createSampleSpecs("spring-framework");
```

Comment:
**Comment maybe outside scope of this PR**

We have a **duplication** in the list of repo folder names between this and the clone samples script.
Perhaps we should extract the clone-samples repo data to a separate file
and generate these specs using that data instead of having to add a "createSampleSpec" line for each new
repo added?

That would be less error prone.